memproxy.go

```diff
@@ -165,18 +165,12 @@ func init() {
 
 // And away we go
 func main() {
-	var l server.ListenArgs
+	var l server.ListenConst
```

Comment:
I had a hard time to imagine that ```Const``` is for "constructor" until I scrolled down in the PR.